apis/server/container_bridge.go

```diff
@@ -101,6 +101,14 @@ func (s *Server) getContainer(ctx context.Context, rw http.ResponseWriter, req *
 		},
 		Mounts:          mounts,
 		NetworkSettings: c.NetworkSettings,
+		Path:            c.Path,
+		Args:            c.Args,
+		ResolvConfPath:  c.ResolvConfPath,
```

Comment:
Can we add some cli test case to lock inspect fields check? :)

Reply:
Yeah, I need to add some test. I will do that in this weekend. While actually I left my laptop charger in the company.